src/lib/Query.ts

```diff
@@ -0,0 +1,6 @@
+export type Query = {
+  startTime?: Date | undefined;
+  endTime?: Date | undefined;
+  contentTopic: string;
+  pageSize?: number | undefined;
```

Comment:
I think these should be
`topic`, `limit`, `before` and `after`

Reply:
we might also want to simplify the encode/decode of dates by using something like:
`beforeMillis` and `afterMillis` and expecting millisecond-timestamps (which are easier to toss around).